.github/workflows/_binary-build-linux.yml

```diff
@@ -12,10 +12,14 @@ on:
         type: string
         description: The build environment
       runs_on:
-          required: false
-          default: linux.12xlarge
-          type: string
-          description: Hardware to run this "build"job on, linux.12xlarge or linux.arm64.2xlarge.
+        required: false
+        default: linux.12xlarge
+        type: string
+        description: Hardware to run this "build"job on, linux.12xlarge or linux.arm64.2xlarge.
+      timeout-minutes:
+        required: false
+        type: string
+        description: timeout for the job
```

Comment:
Here default value needs to be provided

Reply:
Thanks Andrey! This solved the syntax issue of "unexpected value:210".